### PR TITLE
Banner CTA Updates - more attention to Intro Quiz CTA

### LINKED ISSUE #169: [5] Home page banner (focus on quiz call to action)
- [x] Move email sign up to previous position (short banner after main banner)
- [x] Focus main banner on an Intro Quiz Call to action

### PR DESCRIPTION
**Pull request by jamiesonreinhard**
closes #169 

moved header banner paragraph to LSAT Cracked Difference and replaced with info about the intro quiz. Updated intro quiz button to larger size with different text and a pulse effect. Moved email lead form to bottom center of banner

### CODE REVIEW

**Comment by jamiesonreinhard** [comment]
@caioertai I reviewed the changes with Sean and he got a screen shot to review with Matt. He likes the changes to the banner text and the pulse effect and larger "Take the Quiz" button. Still debating what to do with the email lead form.

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -10,3 +10,7 @@
 @import "components/index";
 @import "pages/index";
 @import "plugins/index";
+
+html {
+  scroll-behavior: smooth;
```
Comment (caioertai) [inline app/assets/stylesheets/application.scss:15]: Nice! I learned about it only a few months ago. I'd probably have forgotten it and used the old way of doing this with JS stuff.

File: app/assets/stylesheets/components/_banner.scss [hunk 2]
```diff
@@ -141,4 +141,29 @@ $line-color: $primary;
...
     animation-timing-function: cubic-bezier(0.1, 0.7, 1.0, 0.1);
   }
+
+  .banner-leads-form {
+    position: absolute;
+    left: 50%;
+    bottom: 75px;
+    transform: translateX(-50%);
+    z-index: 5;
+  }
```
Comment (caioertai) [inline app/assets/stylesheets/components/_banner.scss:151]: I tried breaking it by reducing the height and didn't manage to. Clean CSS.

It does break on phones tho. But it's not our priority and the current style is better overall, so I'm just making a note of this and moving on. Don't worry about it unless you find a quick fix. (I'd start with just hiding Matt's caption).
image

File: app/views/pages/_banner.html.erb [hunk 3]
```diff
@@ -4,13 +4,20 @@
       <div class="col-12 col-lg-6">
         <div class="banner-text-main position-relative" style="z-index: 9">
           <h1 class="mb-2">Master the Skills,<br>Master the LSAT</h1>
-          <p class="d-none d-md-block">LSAT Cracked is the only LSAT course that directly identifies the exact skills you're currently missing that an LSAT writer cares about. Stop wasting time repeating the same mistakes, and start unlocking your future career.</p>
-          <%= render "mail_leads/form", mail_lead: @mail_lead %>
-          <%= link_to "Test Your Skills With A Quiz", root_path(anchor: "intro-quiz"), class: "btn btn-primary mt-3" %>
+          <% if @current_user.nil? %>
```
Comment (caioertai) [inline app/views/pages/_banner.html.erb:7]: I'm surprised `@current_user` even exists. I'd rather change this to just `current_user` tho to make a more consistent use of devise's helper.

**Review by caioertai — COMMENTED** [review]
Just some notes. I know it's a work in progress. :postal_horn:

**Comment by jamiesonreinhard** [comment]
Screen Shot 2021-03-31 at 9 13 47 AM

**Pull request merged**